docs/notebooks/application_biology.ipynb

```diff
@@ -150,9 +151,9 @@
    "source": [
     "## Infers cells' ancestors\n",
     "\n",
-    "Cells that are identified as induced Pluripotent Stem Cells (iPSCs) at day 18  are selected, and its ancestors at each previous time points, from day 18 to day 0, are inferred. To do so, the method ```apply_transport_from_potentials``` is used, which enables to perform the pull back operation without instantiating directly the transport matrix. \n",
+    "Cells that are identified as induced Pluripotent Stem Cells (iPSCs) at day 18  are selected, and its ancestors at each previous time points, from day 18 to day 0, are inferred. To do so, the method {meth}`~ott.geometry.pointcloud.PointCloud.apply_transport_from_potentials` is used, which enables to perform the pull back operation without instantiating directly the transport matrix. \n",
```

Comment:
edit suggestion, given the new API: it seems to me we should, rather than store

`dict_results[day] = [geom, out.f, out.g]`

simply do

`dict_results[day] = out`

and then have 
```
cell_distribution = dict_results[reverse_days[i + 1]].apply(cell_distribution_ipsc[day], axis=1)
```
instead of

```
cell_distribution = geom.apply_transport_from_potentials(
        f, g, cell_distribution_ipsc[day], axis=1
    )
```
then we can remove the reference to `apply_from_potentials`, which is too low level.